packages/amplify-category-notifications/src/__tests__/channel-APNS.test.ts

```diff
@@ -1,15 +1,21 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable import/namespace */
+/* eslint-disable spellcheck/spell-checker */
+/* eslint-disable jest/valid-expect-in-promise */
+/* eslint-disable jest/no-conditional-expect */
+import { $TSAny } from 'amplify-cli-core';
+import * as configureKey from '../apns-key-config';
+import * as configureCertificate from '../apns-cert-config';
+import * as channelAPNS from '../channel-APNS';
+
 const inquirer = require('inquirer');
 const mockirer = require('mockirer');
```

Comment:
We should look at deprecating this package : https://www.npmjs.com/package/mockirer

Reply:
agree, but I don't think this should be part of this conversion